python/dgl/heterograph.py

```diff
@@ -1981,36 +2709,50 @@ def predecessors(self, v, etype=None):
         return self._graph.predecessors(self.get_etype_id(etype), v)
```

Comment:
what will happen if user give 'follows' but there are two edge types called 'follows'.

Reply:
It will raise an error as follows "DGLError: Edge type "xxx" is ambiguous. Please use canonical edge type in the form of (srctype, etype, dsttype)"